mypy.ini

```diff
@@ -0,0 +1,107 @@
+[mypy]
+check_untyped_defs = True
+strict_optional = True
+[mypy-docs.macros]
+ignore_errors = True
+[mypy-up42.asset]
+ignore_errors = False
```

Comment:
We don't need to add a module with False, it's default. On each step we're supposed to remove a module and the corresponding test module from this file